src/include/homestore/btree/detail/btree_internal.hpp

```diff
@@ -242,7 +242,8 @@ struct BtreeConfig {
     uint8_t m_suggested_min_pct{30};
     uint8_t m_split_pct{50};
     uint32_t m_max_merge_nodes{3};
-    uint32_t m_rebalance_turned_on{false};
+    bool m_rebalance_turned_on{false};
```

Comment:
Why are both false by default ?

Reply:
I set `m_merge_turned_on` to be true. If the number of calculated new nodes is greater than if  number of current node, the value of rebalance determines that merge needed or not, while m_merge_turned_on is to enable the merge. Later we probably need to introduce a flag to enable/disable the merge during runtime if it is costly.